libs/ui/src/modal.test.tsx

```diff
@@ -33,22 +33,7 @@ describe('Modal', () => {
     render(<Modal content="Do you want to do the thing?" centerContent />);
 
     const modal = screen.getByRole('alertdialog');
-    expect(modal).toMatchInlineSnapshot(`
-      <div
-        aria-label="Alert Modal"
-        aria-modal="true"
-        class="sc-gsDJrp izKgnL ReactModal__Content _"
-        data-testid="modal"
-        role="alertdialog"
-        tabindex="-1"
-      >
-        <div
-          class="sc-hKwCoD bNJKkA"
-        >
-          Do you want to do the thing?
-        </div>
-      </div>
-    `);
+    expect(modal).toMatchSnapshot();
```

Comment:
Should we remove this snapshot?

Reply:
yes, this is covered by style assertions.